### PR TITLE
Ignore FuseGraph Call on Windows

### PR DESCRIPTION
**Pull request by eellison**
Fusion is NYI implemented on Windows, so ignore FuseGraph call instead of failing.

### CODE REVIEW

File: test/test_jit.py [hunk 1]
```diff
@@ -470,6 +470,22 @@ def forward(self, x):
         self.assertExportImport(trace, (t,) + tuple(model.parameters()))
         self.assertExpectedONNXGraph(trace)
 
+    @unittest.skipIf(not IS_WINDOWS, "Testing Fuse skipped on windows")
```
Comment (zou3519) [inline test/test_jit.py]: Can we remove all the other @unittest.skipIf(IS_WINDOWS) from the other fuser tests now?

Comment (zou3519) [reply test/test_jit.py]: edit: Oh I guess we can't if there are expect tests...

File: torch/csrc/jit/passes/graph_fuser.cpp [hunk 2]
```diff
@@ -905,9 +905,14 @@ struct GraphFuser {
...
 
 void FuseGraph(std::shared_ptr<Graph>& graph) {
+  // NYI on Windows
+  #ifndef _WIN32
+
   GraphFuser(graph->block()).run();
   // After FuseGraph some common subexpressions may come back
   EliminateCommonSubexpression(graph);
+
+  #endif
```
Comment (apaszke) [inline torch/csrc/jit/passes/graph_fuser.cpp]: Why is that needed again? It would be better to make the pass a no-op if the fuser doesn't work.

**Review by apaszke — DISMISSED** [review]

File: torch/csrc/jit/passes/graph_fuser.cpp (hunk 2 above)
Comment (eellison) [reply torch/csrc/jit/passes/graph_fuser.cpp]: Isn't it currently making the pass a no-op?

I don't have the full context on why fuser doesn't work on windows.

Comment (apaszke) [reply torch/csrc/jit/passes/graph_fuser.cpp]: Never mind. I confused this code with that in `graph_executor.cpp`, which looks exactly the same now.

**Review by apaszke — APPROVED** [review]
